extensions/wikia/TemplateClassification/TemplateClassification.i18n.php

```diff
@@ -4,12 +4,44 @@
 $messages['en'] = [
 	'template-classification-type-unclassified' => 'Unclassified',
 	'template-classification-type-infobox' => 'Infobox',
+	'template-classification-description-infobox' => 'Displays the most essential snippets of information about an article in a box.',
+
 	'template-classification-type-navbox' => 'Navbox',
+	'template-classification-description-navbox' => 'Provides a list of organized links for readers to navigate to similar articles.',
+
 	'template-classification-type-quote' => 'Quote',
+	'template-classification-description-quote' => 'Highlights notable snippets of text such as lyrics, book passages, or quotes.',
+
+	'template-classification-type-notice' => 'Notice',
+	'template-classification-description-notice' => 'Also known as tophats or alerts, these templates notify the reader about the status of the article itself, such as stub, disambiguation, or spoiler.',
+
+        'template-classification-type-reference' => 'Citation or reference',
+        'template-classification-description-reference' => 'Organizes and standardizes how sources of content on an article page are annotated.',
+
+        'template-classification-type-media' => 'Image, video, or gallery',
+        'template-classification-description-media' => 'This template will standardize or modify the display of one or more images or videos.',
+
+        'template-classification-type-data' => 'Data',
+        'template-classification-description-data' => 'This template is used to display data on an article page.',
+
+        'template-classification-type-design' => 'Design',
+        'template-classification-description-design' => 'This template is used to design or structure article pages.',
+
+        'template-classification-type-navigation' => 'Navigation',
+        'template-classification-description-navigation' => 'This template is used for navigation.',
+
+        'template-classification-type-nonarticle' => 'Non-article',
+        'template-classification-description-nonarticle' => 'This template is not used on article pages.',
+
+        'template-classification-type-unknown' => 'Unknown',
+        'template-classification-description-unknown' => "I don't know.",
```

Comment:
Some whitespace issues, the messages above are all indented with spaces instead of tabs.